source/02_language_guide/23_Opaque_Types.md

```diff
@@ -246,4 +246,4 @@ print(type(of: twelve))
 // 输出 "Int"
 ```
 
-`twelve` 的类型可以被推断出为 `Int`， 这说明了类型推断适用于不透明类型。在 `makeOpaqueContainer(item:)` 的实现中，底层类型是不透明集合 `[T]`。在上述这种情况下，`T` 就是 `Int` 类型，所以返回值就是整数数组，而关联类型 `Item` 也被推断出为 `Int`。`Container` 协议中的 `subscipt` 方法会返回 `Item`，这也意味着 `twelve` 的类型也被能推断出为 `Int`。
```

Comment:
I have actually no idea why GitHub commits this change as well lmao

Reply:
Maybe there's an invisible character added or removed, seems fine.